frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -460,3 +470,79 @@ action.GitMachete.CheckoutParentAction.undefined.branch-name=Branch (''{0}'') do
+
+# Traverse
+
+action.GitMachete.BaseTraverseAction.description=Traverse
+action.GitMachete.BaseTraverseAction.description.empty-layout=Cannot traverse an empty branch layout
+
+action.GitMachete.BaseTraverseAction.dialog.traverse-approval.title=Traverse Info
+action.GitMachete.BaseTraverseAction.dialog.traverse-approval.text.HTML=<html>\
+  This action triggers an interactive walk over the branches and suggests actions (pushes/pulls/rebases) \
+  necessary to achieve both child-to-parent and local-to-remote synchronization.<br><br>\
+  Do you want to proceed?\
+  </html>
+
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.text.HTML=\
+  <html>\
+    <p>The branch <b>{0}</b> is diverged from remote <b>{1}</b>, and it is <b>{2}</b> remote.</p> \
+    <p>Please choose the preferred action:</p>\
+  </html>
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.title=Resolve Divergence from Remote
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.newer-than-remote=newer than
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.older-than-remote=older than
+
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.do-not-resolve-option=Do not sync
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.do-not-resolve-option.comment=Note: the remote and local branches will remain intact; traverse will continue
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.force-push-option=Force push
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.force-push.comment=Warning: you will lose the remote commits
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.reset-option=Reset to remote
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.reset-on-remote.comment=Warning: you will lose the local commits
+action.GitMachete.BaseTraverseAction.dialog.cancel-traverse=Quit the interactive traverse
+
+action.GitMachete.BaseTraverseAction.dialog.out-of-sync-to-parent.title=Resolve Out of Sync to Parent
+action.GitMachete.BaseTraverseAction.dialog.out-of-sync-to-parent.text.HTML=\
+  <html>\
+    <p>The child branch <b>{0}</b> is out of sync to the parent branch <b>{1}</b>.</p><br>\
+    <p>Do you want to <b>rebase</b> branch <b>{0}</b> branch onto <b>{1}</b>?</p>\
+  </html>
+
+action.GitMachete.BaseTraverseAction.dialog.fork-point-off.title=Resolve In Sync to Parent but Fork Point Off
+action.GitMachete.BaseTraverseAction.dialog.fork-point-off.text.HTML=\
+  <html>\
+    <p>The child branch <b>{0}</b> is in sync to the parent branch <b>{1}</b> \
+    but the range of commits belonging to <b>{0}</b> is uncertain (the fork point is off).</p><br>\
+    <p>Do you want to <b>rebase</b> branch <b>{0}</b> branch onto <b>{1}</b>?</p>\
+  </html>
+
+action.GitMachete.BaseTraverseAction.dialog.merged-to-parent.title=Resolve Merged to Parent
```

Comment:
Should there be a `Git Machete Traverse -` prefix in the title? As with `Git Machete Traverse - Pull` etc. below

Reply:
good point 👍  to reflect that we are in the scope of the ongoing action

Reply:
however I am afraid that it may be too long for some cases (e.g. `Git Machete Traverse - Resolve In Sync to Parent but Fork Point Off`)

Reply:
<img width="623" alt="image" src="https://user-images.githubusercontent.com/19799111/209220895-3b8e59e1-645d-4db2-a043-989bde501363.png">

oh, it works just fine ! cool